apps/condo/domains/marketplace/access/Invoice.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Generated by `createschema marketplace.Invoice 'number:Integer; property:Relationship:Property:PROTECT; unitType:Text; unitName:Text; accountNumber:Text; toPay:Decimal; items:Json; contact?:Relationship:Contact:SET_NULL; client?:Relationship:User:SET_NULL; clientName?:Text; clientPhone?:Text; clientEmail?:Text'`
+ */
+
+const { get, map } = require('lodash')
```

Comment:
After refactor below you can remove map import from this line
```suggestion
const { get } = require('lodash')
```

Reply:
done